docs: add chainsaw capabilities to quick start guide (#973)
kyverno/chainsaw: website/docs/quick-start.md

- Skipped tests 0
Done.
```
### Chainsaw Charaterstics 

> **Resource diff in assertion failures:** Chainsaw now offers detailed resource diffs upon assertion failures. This feature provides a clear view of discrepancies between expected and actual resource configurations, aiding testers in quickly identifying and resolving issues during debugging.

Reviewer: ```suggestion
**Resource diff in assertion failures:** Chainsaw offers detailed resource diffs upon assertion failures. It provides a clear view of discrepancies between expected and actual resource.
```
